cEP-0019.md

```diff
@@ -0,0 +1,483 @@
+
+##### Display
+
+Review comments that need meta-review will be displayed on a separate column
+on gh-board. There are two approaches:
+
+1. A basic version is to store the identifier of the PR comments which
+   do not have meta-reviews and gh-board links to the PR review comment
+   in GitHub PR web app. Doing it on GitHub has the benefit of letting users
+   have their usual tools, e.g.
+   [refined-github](https://github.com/sindresorhus/refined-github).
+
+2. In the ultimate version, gh-board would show the review comments.
+   In the future, people could do meta-reviews directly on gh-board. This also
+   provides an approach to doing meta-reviews anonymously.
+
+#### Performance Requirements
+
+Currently,
+[fetch-issues.js](https://github.com/coala/gh-board/blob/46f1ad5ce83d5d72aa95ea00703b92c2fc38b2dc/script/fetch-issues.js)
+pre-fetches issues and pull requests to speed up gh-board performance and
+save users' API hits. Meta-reviews are also to be fetched during this procedure.
+
+#### Other Features
+
+The GSoC project provides a blueprint for future enhancement. Additional
+features could be implemented to make the meta-review system more
+sophisticated:
+
+1. A basic ranking list will be built first. In the best case, we should
+   offer an overall ranking list as well as a recent ranking list.
+
+2. Offer an option for users to do meta-reviews directly on gh-board. In the
+   future users can do meta-reviews on gh-board anonymously (currently, there
+   is no way for people to do anonymous meta-reviews on GitHub page). Note
+   that people may need context to do meta-reviews. The diff hunk to which
+   the comment applies needs to be displayed on gh-board so that people have
+   a basic context to do meta-reviews directly on gh-board.
+
+3. Display visitor’s own meta-reviews if the visitor logs in. People may want
+   to see meta-reviews they have done before.
+
+4. Provide notifications of meta-reviews to a user if they receive any
+   meta-review. Suppose I ask for someone's agreement and they use 👍 to
+   show their agreement, but I wouldn't receive any notification. I have to
+   check that page regularly. gh-board is a nice place to provide notifications
+   of meta-reviews.
+
+5. There is a potential for reverted commits to be meta-reviewed. A reverted
+   commit should trigger a new meta-review of the original PR, as time may
+   have proved that the merge was incorrect, or maybe the merge was a
+   self-aware time-sensitive workaround, and is no longer needed.
+
+6. An important technical detail of meta-review in coala on GitHub is that
+   the context disappears on an amend push, so meta-review by maintainers after
+   the merge may be without the necessary context to be properly evaluated.
+   However, if the commit SHA is mentioned, GitHub will keep the commit even
+   if it is no longer in the PR. A gitmate plugin is needed in the future to
+   make meta-reviews by maintainers more effective.
```

Comment:
A gitmate plugin is needed ... to ... mention any SHAs which were reviewed, so that meta-reviews by maintainers will have more context and be more effective.